static/src/javascripts/projects/common/modules/commercial/comment-adverts.js

```diff
@@ -44,6 +44,7 @@ define([
             !$adSlotContainer.length ||
             !config.switches.discussion ||
             !identityApi.isUserLoggedIn() ||
+            (config.page.section === 'childrens-books-site' || config.page.shouldHideAdverts) ||
```

Comment:
Probably it is, but this is how we usually test sensitive pages (those two values together). I think it is more visually appealing to leave it as it is now.